src/components/application_manager/test/application_helper_test.cc

```diff
@@ -157,6 +157,8 @@ TEST_F(ApplicationHelperTest, RecallApplicationData_ExpectAppDataReset) {
   cmd[strings::msg_params][strings::menu_id] = menu_id;
   cmd[strings::msg_params][strings::interaction_choice_set_id] = choice_set_id;
 
+  smart_objects::SmartObjectSPtr message_to_hmi;
```

Comment:
Please, replace this line before line 209 where you have next call `EXPECT_CALL(*mock_message_helper_, CreateMessageWithFunctionID(_, hmi_apis::FunctionID::Navigation_UnsubscribeWayPoints)).WillOnce(Return(message_to_hmi));`

Reply:
Done in commit 16f31bfe48381f49e1fee07fb082941f50c21741